dataflux_pytorch/lightning/gcs_filesystem.py

```diff
@@ -4,36 +4,44 @@
 from pathlib import Path
 from typing import Generator, Optional, Union
 
+import torch.distributed as dist
 from dataflux_core import user_agent
-from google.cloud import storage
-from torch.distributed.checkpoint import FileSystemReader, FileSystemWriter
-
 from dataflux_pytorch.dataflux_checkpoint import DatafluxCheckpointBuffer
 from dataflux_pytorch.lightning.path_utils import parse_gcs_path
+from google.cloud import storage
+from torch.distributed.checkpoint import FileSystemReader, FileSystemWriter
 
 
 class GCSFileSystem():
 
     def __init__(
         self,
         project_name: str,
+        debug: bool,
         storage_client: Optional[storage.Client] = None,
     ):
         self.project_name = project_name
         self.storage_client = storage_client
         if not storage_client:
             self.storage_client = storage.Client(project=self.project_name)
         user_agent.add_dataflux_user_agent(self.storage_client)
+        self.debug = debug
 
     @contextmanager
     def create_stream(self, path: Union[str, os.PathLike],
                       mode: str) -> Generator[io.IOBase, None, None]:
         bucket, path = parse_gcs_path(path)
         blob = self.storage_client.bucket(bucket).blob(path)
         if mode == "wb":  # write mode.
+            print(
```

Comment:
Are these statements not intended to be behind the debug flag?

Reply:
Added debug flag